x-pack/plugins/endpoint/server/plugin.ts

```diff
@@ -34,9 +36,16 @@ export class EndpointPlugin
       EndpointPluginStartDependencies
     > {
   private readonly logger: Logger;
+  private readonly endpointAppContextService: EndpointAppContextService;
   constructor(private readonly initializerContext: PluginInitializerContext) {
     this.logger = this.initializerContext.logger.get('endpoint');
+    this.endpointAppContextService = new EndpointAppContextService();
```

Comment:
nit: I think you could do the initialization outside of the constructor on a single line right?

`private readonly endpointAppContextService = new EndpointAppContextService();`